mycli/main.py

```diff
@@ -1098,16 +1093,22 @@ def cli(database, user, host, port, socket, password, dbname,
             else:
                 click.secho(alias)
         sys.exit(0)
+
     if list_ssh_config:
```

Comment:
Side note: the list_ssh_config feature is unreliable if the SSH config file uses certain features such as Match.  Listing the hosts is not really in the scope of a tool such as mycli, and we ought to consider removing it.